met-api/src/met_api/services/contact_service.py

```diff
@@ -11,13 +12,20 @@ class ContactService:
     def get_contact_by_id(contact_id):
         """Get contact by id."""
         contact_record = Contact.get_contact_by_id(contact_id)
-        return ContactSchema().dump(contact_record)
+        contact = ContactSchema().dump(contact_record)
+        contact['avatar_url'] = ObjectStorageService.get_url(contact.get('avatar_filename', None))
```

Comment:
i forgot whats in the O.get_url() does ;just thinking will it be faster if we store the full url in the DB and avoid this logic? We can discuss this and weigh each approach

Reply:
We can discuss this. The reasoning behind it is that this will work even if host changed.